src/coreclr/interpreter/compiler.cpp

```diff
@@ -1278,7 +1278,7 @@ void InterpCompiler::PatchInitLocals(CORINFO_METHOD_INFO* methodInfo)
 }
 
 // Adds a conversion instruction for the value pointed to by sp, also updating the stack information
-void InterpCompiler::EmitConv(StackInfo *sp, StackType type, InterpOpcode convOp)
+InterpInst * InterpCompiler::EmitConv(StackInfo *sp, StackType type, InterpOpcode convOp)
```

Comment:
It looks like this change is not needed, I don't see the return value being used anywhere.

Reply:
The return value is used in the EmitConv caller to store the helper ftn data item into data[0] for the fp conversions

Reply:
Ah, I've missed that there is the -> after the call. It is different from all other places where we so something with the just created instruction though. We use the `m_pLastNewIns` instead, so I am not sure if introducing a new pattern won't be confusing.

Reply:
I would prefer consistency with the other code here.

Reply:
I will change it to match and just add a comment that this behavior is load-bearing.